BUILD.bazel

```diff
@@ -185,77 +185,235 @@ swift_cc_test_library(
     deps = ["@gtest"],
 )
 
+# Common test dependencies
+TEST_DEPS = [
+    ":albatross",
+    ":serialize-testsuite",
+    "@zlib",
+    "@gtest//:gtest_main",
+    "@suitesparse//:spqr",
+]
+
+TEST_COMMON = [
```

Comment:
Can you extract a library out of that, e.g. `swift_cc_test_library`?

Reply:
I think it would not really change anything, except that the header files are modeled properly. But I think, it's also fine this way.

Reply:
I am happy to fix it if it's meaningful to have a lib of just headers.  Please have a look at the latest version.